robot/packages/cloud_package_manager.go

```diff
@@ -171,22 +165,24 @@ func (m *cloudManager) Sync(ctx context.Context, packages []config.PackageConfig
 
 		m.logger.Debugf("Downloading from %s", sanitizeURLForLogs(resp.Package.Url))
 
-		nonEmptyPaths := make([]string, 0)
-		if p.Type == config.PackageTypeModule {
-			matchedModules := m.modulesForPackage(p, modules)
-			if len(matchedModules) == 1 {
-				nonEmptyPaths = append(nonEmptyPaths, matchedModules[0].ExePath)
-			}
-			if len(matchedModules) > 1 {
-				m.logger.CWarnf(ctx, "package %s matched %d > 1 modules, not doing entrypoint checking", p.Name, len(matchedModules))
-			}
-		}
-
 		// download package from a http endpoint
-		err = installPackage(ctx, m.logger, m.packagesDir, resp.Package.Url, p, nonEmptyPaths,
-			func(ctx context.Context, url, dstPath string) (string, error) {
-				_, contentType, err := m.downloadFileFromGCSURL(ctx, url, dstPath, m.cloudConfig.ID, m.cloudConfig.Secret)
-				return contentType, err
+		err = installPackage(ctx, m.logger, m.packagesDir, resp.Package.Url, p,
+			func(ctx context.Context, url, dstPath string) (string, string, error) {
+				statusFile := packageSyncFile{
+					PackageID:       p.Package,
+					Version:         p.Version,
+					ModifiedTime:    time.Now(),
+					Status:          syncStatusDownloading,
```

Comment:
wishlist -- use state graphs for enums like this so we can warn on unexpected transitions + more clearly model the system

(not actionable for this PR though)